org.jacoco.core/src/org/jacoco/core/internal/analysis/filter/SyntheticFilter.java

```diff
@@ -44,4 +50,14 @@ public void filter(final MethodNode methodNode,
 				methodNode.instructions.getLast());
 	}
 
+	private static final Pattern ASPECTJ_AROUND_BODY_PATTERN = Pattern.compile(".*_aroundBody\\d+");
+
+	/**
+	 * @see AspectjFilter
+	 */
+	private boolean isHandledByAspectJFilter(MethodNode methodNode) {
```

Comment:
Can't find anything about `_aroundBody` in `AspectjFilter`.

Reply:
When a method `foo()` is adivsed with an around advice, the bytecode of the method implementation is moved to `foo_aroundBody0()`. This method has the `ACC_SYNTHETIC` flag, but must not be filtered because it contains actual user code.

I'll add a comment to this method to make this more clear.